File: src/lambda_quine.js
const stanza = `

exports.handler = (event, context, cb) => {
  const backtick = String.fromCharCode(96);
  const quine = 'const stanza = '
    + backtick + stanza + backtick + ';'
    + stanza;
  const response = {
    statusCode: 200,
    body: quine,
  };
  cb(null, response);
};
`;

exports.handler = (event, context, cb) => {
  const backtick = String.fromCharCode(96);
  const quine = 'const stanza = '
    + backtick + stanza + backtick + ';'
    + stanza;
  const response = {
    statusCode: 200,
    body: quine,
  };
  cb(null, response);
};

File: src/lambda_quine.py
stanza = """


def lambda_handler(event, context):
    marker = chr(34) * 3
    quine = 'stanza = ' + marker + stanza + marker + stanza
    return {'statusCode': '200',
            'body': quine}
"""


def lambda_handler(event, context):
    marker = chr(34) * 3
    quine = 'stanza = ' + marker + stanza + marker + stanza
    return {'statusCode': '200',
            'body': quine}
